torchvision/models/optical_flow/__init__.py

```diff
@@ -0,0 +1 @@
+from ._raft.raft import RAFT, raft, raft_small
```

Comment:
We should expose the building blocks as well, stuff like `ResidualBlock`, `FeatureEncoder`, etc.

Should we expose these in a `models.optical_flow.raft` namespace?

I remember @datumbox mentioning a few issues when we have both a module and a function with the same name. Ideally, I'd like to keep the `raft()` name for the function builder, but I'm happy to get your thoughts on this

Reply:
Thanks for the feedback 

> Can you talk about the choice to create a second private submodule _raft? Why is that?

I wrote it like this out of habit but I had no intention of keeping it this way

Instead of renaming `raft()` to `raft_large()`, would it be OK instead to rename the `.py` file to something like `raft_core.py` ? Or `raft_implem.py`?

We would have `raft()`, `raft_small` and `RAFT` available from `torchvision.models.optical_flow` and then we would be able to access the other building blocks like `ResidualBlock` in `torchvision.models.optical_flow.raft_core`  -- I would make sure to exclude them from `__all__`.

Would that be OK?

Reply:
Though calling it `raft_large` would make it consistent with other small/large models, I can see that on the paper they don't actually call it large. They do use `Raft-s` or "small" for the small version, so that's canonical at least. So I understand why you want to avoid naming it like that.

Renaming the file should be OK but @pmeier should confirm. Though I'm not sure what the actual name should be. We typically name the files after their algorithm and we let the model builder have extra info on the variant. 

@fmassa thoughts on this?

Reply:
OK, I'll use `raft.py` and `raft_large()` for now to move forward. We can revisit later if needed, thanks for the input